api/api.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2015 The Hugo Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api
+
+import (
+	"github.com/spf13/hugo/commands"
+	"github.com/spf13/viper"
+)
+
+func Run(flags []string) {
+	commands.Execute(flags)
+}
```

Comment:
Sure, you're right. Let's start with something simple, but it can be improved in the near future. The api could be also extended for the other commands, for example.